mln/models/dynamic/module.py

```diff
@@ -48,10 +49,12 @@ def save(self, *args, **kwargs):
 	def _calc_yield_info(self):
 		"""Calculate the yield of this module (how many items you can harvest), as well as the time and clicks that remain."""
 		if self._needs_setup() and not self.is_setup:
-			return self.yield_since_last_harvest, 0, 0
+			yield_info = get_or_none(ModuleHarvestYield, item_id=self.item_id)
+			limit = float("inf") if yield_info is None else yield_info.max_yield
+			return min(limit, self.yield_since_last_harvest), timedelta(seconds=0), 0
```

Comment:
Couldn't this just be
```python
yield = self.yield_since_last_harvest if yield_info is None else min(self.yield_since_last_harvest, yield_info.max_yield)
return yield, timedelta(seconds=0), 0
```
?
Comparing with infinity seems a bit hacky and when either solution is two lines anyway we might just use the one that avoids it.

Reply:
```suggestion
			yield = self.yield_since_last_harvest if yield_info is None else min(self.yield_since_last_harvest, yield_info.max_yield)
			return yield, timedelta(seconds=0), 0
```